core/platform-common/src/main/java/org/sunbird/keys/JsonKey.java

```diff
@@ -633,10 +635,19 @@ public final class JsonKey {
   public static final String MANDATORY_FIELDS_EXISTS = "mandatoryFieldsExists";
   public static final String OSID = "osid";
   public static final String DISABLE_MULTIPLE_ORG_ROLE = "sunbird_disable_multiple_org_role";
-  public static final String PASSWORD_RESET_LOGIN_PAGE_URL = "sunbird_password_reset_login_page_url";
+  public static final String PASSWORD_RESET_LOGIN_PAGE_URL =
+      "sunbird_password_reset_login_page_url";
   public static final String CLOUD_SERVICE_PROVIDER = "sunbird_cloud_service_provider";
   public static final String EXHAUST_ENCRYPTION_KEY = "exhaustEncryptionKey";
   public static final String CLOUD_SERVICE_CONTAINER = "sunbird_content_cloud_storage_container";
+  public static final String DEFAULT = "default";
+  public static final String DATA_SECURITY_POLICY = "dataSecurityPolicy";
+  public static final String JOB = "job";
+  public static final String LEVEL_CHAR = "L";
+  public static final String LEVEL_1 = "L1";
+  public static final String LEVEL_2 = "L2";
+  public static final String LEVEL_3 = "L3";
+  public static final String LEVEL_4 = "L4";
```

Comment:
Why don't we use 1,2,3,4 directly as integer values?

Reply:
L1, L2, L3 and L4 are the agreed values in design document as 4 levels of setting.